<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-72NPBGM" failures="2" tests="8" name="Test" time="36.083" errors="0" timestamp="2022-01-11T17:49:48 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.686">
    <failure type="java.lang.AssertionError" message="expected [validation succeeded] but found [validation failed]">
      <![CDATA[java.lang.AssertionError: expected [validation succeeded] but found [validation failed]
at pageObjects.Base_PO.waitForAlert_And_ValidateText(Base_PO.java:71)
at stepDefinitions.LoginSteps.i_should_be_presented_with_the_following_login_validation_message(LoginSteps.java:49)
at ✽.I should be presented with the following login validation message validation succeeded(classpath:features/Login.feature:24)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="11.644"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.142"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.102"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.259"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.978">
    <failure type="java.lang.AssertionError" message="expected [validation succeeded] but found [validation failed]">
      <![CDATA[java.lang.AssertionError: expected [validation succeeded] but found [validation failed]
at pageObjects.Base_PO.waitForAlert_And_ValidateText(Base_PO.java:71)
at stepDefinitions.LoginSteps.i_should_be_presented_with_the_following_login_validation_message(LoginSteps.java:49)
at ✽.I should be presented with the following login validation message validation succeeded(classpath:features/Login.feature:24)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.88"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.069"/>
</testsuite> <!-- Test -->
